codegen/src/main/java/com/paypal/selion/elements/MobileSeLionElementList.java

```diff
@@ -0,0 +1,120 @@
+|  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance     |
+|  with the License.                                                                                                  |
+|                                                                                                                     |
+|  You may obtain a copy of the License at                                                                            |
+|                                                                                                                     |
+|       http://www.apache.org/licenses/LICENSE-2.0                                                                    |
+|                                                                                                                     |
+|  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed   |
+|  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for  |
+|  the specific language governing permissions and limitations under the License.                                     |
+\*-------------------------------------------------------------------------------------------------------------------*/
+
+package com.paypal.selion.elements;
+
+import com.paypal.selion.plugins.GUIObjectDetails;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class MobileSeLionElementList extends AbstractSeLionElementList {
+    public static final String UIAUTOMATION_ELEMENT_CLASS = "com.paypal.selion.platform.mobile.elements";
+
+    public static MobileSeLionElementList Button =
+            new MobileSeLionElementList(UIAUTOMATION_ELEMENT_CLASS, "Button", true);
+    public static MobileSeLionElementList Element =
+            new MobileSeLionElementList(UIAUTOMATION_ELEMENT_CLASS, "Element", true);
+    public static MobileSeLionElementList Image =
+            new MobileSeLionElementList(UIAUTOMATION_ELEMENT_CLASS, "Image", true);
+    public static MobileSeLionElementList Label =
+            new MobileSeLionElementList(UIAUTOMATION_ELEMENT_CLASS, "Label", true);
+    public static MobileSeLionElementList List =
+            new MobileSeLionElementList(UIAUTOMATION_ELEMENT_CLASS, "List", true);
+    public static MobileSeLionElementList Picker =
+            new MobileSeLionElementList(UIAUTOMATION_ELEMENT_CLASS, "Picker", true);
+    public static MobileSeLionElementList Slider =
+            new MobileSeLionElementList(UIAUTOMATION_ELEMENT_CLASS, "Slider", true);
+    public static MobileSeLionElementList Switch =
+            new MobileSeLionElementList(UIAUTOMATION_ELEMENT_CLASS, "Switch", true);
+    public static MobileSeLionElementList TextField =
+            new MobileSeLionElementList(UIAUTOMATION_ELEMENT_CLASS, "TextField", true);
+
+    public static MobileSeLionElementList BASE_CLASS =
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The field name indicates a constant but its modifiers do not](https://www.codacy.com/app/SeLion/paypal-SeLion/file/2360069938/issues/source?bid=2928425&fileBranchId=3219780#l46)